### PR TITLE
Add missing option in the CMake for Pose2 Expmap

### PR DESCRIPTION
**Pull request by ProfFan**
This PR adds the missing option in CMake, the code exists already just not exposed.

### CODE REVIEW

**Review by varunagrawal — APPROVED** [review]
LGTM, but CI fails?

File: cmake/HandleGeneralOptions.cmake [hunk 1]
```diff
@@ -30,6 +30,7 @@ option(GTSAM_ALLOW_DEPRECATED_SINCE_V43     "Allow use of methods/functions depr
 option(GTSAM_SUPPORT_NESTED_DISSECTION      "Support Metis-based nested dissection" ON)
 option(GTSAM_TANGENT_PREINTEGRATION         "Use new ImuFactor with integration on tangent space" ON)
 option(GTSAM_SLOW_BUT_CORRECT_BETWEENFACTOR "Use the slower but correct version of BetweenFactor" OFF)
+option(GTSAM_SLOW_BUT_CORRECT_EXPMAP        "Use slower but correct expmap"  ON)
```
Comment (varunagrawal) [inline cmake/HandleGeneralOptions.cmake:33]: I think this should be OFF by default. That fixes the CI.

Comment (ProfFan) [reply cmake/HandleGeneralOptions.cmake:33]: Yup, a lot of stuff in CI assumes otherwise...

**Comment by varunagrawal** [comment]
Looks great! Merging.

**Pull request merged**